src/logic/graphSamples.ts

```diff
@@ -12,22 +12,24 @@ const samples: { [key: string]: any } = {
     ],
     string: '0 2 6 0 0 0 0\n2 0 0 5 0 0 0\n6 0 0 8 0 0 0\n0 5 8 0 10 15 0\n0 0 0 10 0 6 2\n0 0 0 15 6 0 6\n0 0 0 0 2 6 0',
   },
-  simple: {
+  medium: {
     graph: [
-      [0, 1, 1, 1], 
-      [1, 0, 1, 1], 
-      [1, 1, 0, 1], 
-      [1, 1, 1, 0],
+      [0, 2, 6, 0, 0],
+      [2, 0, 0, 5, 0],
+      [6, 0, 0, 8, 0],
+      [0, 5, 8, 0, 10],
+      [0, 0, 0, 10, 0],
     ],
-    string: '0 1 1 1\n1 0 1 1\n1 1 0 1\n1 1 1 0',
+    string: '0 2 6 0 0\n2 0 0 5 0\n6 0 0 8 0\n0 5 8 0 10\n0 0 0 10 0',
```

Comment:
There should be a function to do this.

Reply:
Right, let's add that the next time when some of us will be adjusting it